features/reisdocumenten.feature

```diff
@@ -89,4 +89,148 @@ Functionaliteit: Raadplegen van reisdocumenten op basis van een reisdocumentnumm
     | reisdocumentnummer           | toelichting                                       |
     | <script>hello world</script> | reisdocumentnummer bevat ongeldige karakters      |
     | abc123AB                     | reisdocumentnummer is minder dan 9 karakters lang |
-    | abcd1234AB                   | reisdocumentnummer is meer dan 9 karakters lang   |
\ No newline at end of file
+    | abcd1234AB                   | reisdocumentnummer is meer dan 9 karakters lang   |
+
+  Rule: een reisdocumentnummer wordt niet opgenomen wanneer Datum einde geldigheid Nederlands reisdocument (35.50) vandaag is of in het verleden ligt
+    - een reisdocument wordt alleen opgenomen wanneer Datum einde geldigheid Nederlands reisdocument leeg is, volledig onbekend is of in de toekommst ligt
```

Comment:
Kunnen we hier ook niet net als bij Bewoning het afvalbakje introduceren?
@fsamwel komt dit voor?

Reply:
@CathyDingemanse ik begrijp je vraag niet. wat zou je in dat afvalbakje willen stoppen?